.github/workflows/xcode-versions-build.yml

```diff
@@ -0,0 +1,74 @@
+name: Multi-Xcode Build
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  build:
+    name: "Build App"
+    runs-on: macos-13
+    strategy:
+      max-parallel: 3
+      matrix:
+        variant: [cocoapods, spm]
+        xcode: [14.3, 14.2, 14.1]
```

Comment:
Why do we do it for 14.2 and 14.1? I would say 14.3.1 should be enough...

Reply:
My reasoning was covering all Xcode versions which can submit a build to the AppStore
Do you think it is safe to say if it builds on the latest Xcode 14 we are OK?

Reply:
I think latest 14 is safe unless we start to build against specific deploy targets / swift versions too (which is food for thought!)